utils.h

```diff
@@ -15,30 +15,16 @@ struct device_type {
     type value;
     std::string name;
 
+    device_type() : value(unknown), name("unknown") {} // Default constructor
```

Comment:
Needed empty initializer here for `device_type device` class member variable in `csi_camera.h`.